docs/reference/reference-deploy/config/master-config-reference.rst

```diff
@@ -250,9 +250,17 @@ The master supports the following configuration settings:
          is started.
 
       -  ``slot_type``: The default slot type assumed when users request resources from Determined
-         in terms of ``slots``. Available values are ``cuda`` and ``cpu``, where 1 ``cuda`` slot is
-         1 GPU and 1 ``cpu`` slot is 1 node. Defaults per partition to ``cuda`` if GPU resources are
-         found within the partition, else ``cpu``.
+         in terms of ``slots``. Defaults to ``cuda``.
+
+         -  ``slot_type: cuda``: One NVIDIA GPU will be requested per compute slot. Partitions which
```

Comment:
I think we are saying that any partitions with GPUs will be represented as a resource pool of slot type cuda, which can be overridden in the experiment configuration.